core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -205,7 +205,7 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         <md-card ng-repeat="exploration in explorationsList | orderBy:sortByFunction:isCurrentSortDescending track by exploration.id"
                  class="oppia-activity-summary-tile oppia-dashboard-card-view-item protractor-test-exploration-dashboard-card"
                  is-private="exploration.status === 'private'">
-          <a ng-href="/create/<[exploration.id]>">
+          <a ng-href="/create/<[exploration.id]>#/gui/Introduction">
```

Comment:
I added the gui/Introduction part because by default each exploration is redirected to that part from the dashboard page.

Fixed the creation loop by appending created=true to the url so that the creation part is not called again when back button is pressed.